### PR TITLE
added diverter commands

### PR DESCRIPTION
**Pull request by dariuszSki**

### CODE REVIEW

**Review by jlin-nf — APPROVED** [review]

File: nfhelp.sh [hunk 1]
```diff
@@ -218,9 +225,14 @@ create_aliases() {
     alias sar-enable="echo 'ENABLED="true"'| sudo tee /etc/default/sysstat"
     alias sar-disable="echo 'ENABLED="false"'| sudo tee /etc/default/sysstat"
     alias sar-status="sudo cat /etc/default/sysstat"
-    alias geneve-enable="sudo tc qdisc add dev ${MYIF%:} clsact && sudo tc filter add dev ${MYIF%:} ingress bpf da obj /opt/netfoundry/ebpf/geneve.o sec sk_skb"
+    alias geneve-enable="sudo tc qdisc add dev ${MYIF%:} clsact && sudo tc filter add dev ${MYIF%:} ingress bpf da obj \$EBPF_HOME/geneve.o sec sk_skb"
     alias geneve-disable="sudo tc qdisc del dev ${MYIF%:} clsact"
     alias geneve-status="sudo tc filter show dev ${MYIF%:} ingress"
+    alias geneve-update="curl -sL /tmp/geneve.file https://api.github.com/repos/r-caamano/tc-ebpf-geneve-decapsulator/releases/latest > /tmp/geneve.file; export URL=\$(cat /tmp/geneve.file | jq -r .assets[].browser_download_url) && export NAME=\$(cat /tmp/geneve.file | jq -r .assets[].name); curl -sL \$URL > /tmp/\$NAME && sudo tar xzf /tmp/\$NAME -C \$EBPF_HOME; rm /tmp/geneve.file /tmp/\$NAME; unset URL NAME"
```
Comment (emoscardini) [inline nfhelp.sh:231]: Before we merge this, let's fork that repo into the netfoundry org.

File: nfhelp.sh [hunk 2]
```diff
@@ -218,9 +225,14 @@ create_aliases() {
...
+    alias diverter-enable="sudo $EBPF_HOME/scripts/tproxy_splicer_startup.sh --initial-setup"
+    alias diverter-disable="sudo $EBPF_HOME/scripts/tproxy_splicer_startup.sh --revert-tproxy"
+    alias diverter-status="sudo tc filter show dev ${MYIF%:} ingress"
+    alias diverter-update="curl -sL /tmp/tproxy.file https://api.github.com/repos/r-caamano/ebpf-tproxy-splicer/releases/latest > /tmp/tproxy.file; export URL=\$(cat /tmp/tproxy.file | jq -r .assets[].browser_download_url) && export NAME=\$(cat /tmp/tproxy.file | jq -r .assets[].name); curl -sL \$URL > /tmp/\$NAME && sudo tar xzf /tmp/\$NAME -C \$EBPF_HOME; rm /tmp/tproxy.file /tmp/\$NAME; unset URL NAME"
```
Comment (emoscardini) [inline nfhelp.sh:235]: Same for this one.

Comment (dariuszSki) [reply nfhelp.sh]: ok, I removed geneve commands and updated the diverter reference. Also, updated CLOUD_HOME_ZITI in ZITI_HOME @emoscardini let me know if this is ok

**Review by emoscardini — CHANGES_REQUESTED** [review]
Looks good now, but we should bump the version number to 1.1.0 & add a change log item.

**Comment by dariuszSki** [comment]
good idea, will add it

**Comment by dariuszSki** [comment]
@emoscardini Added the changelog file, is that what you had in mind. I still don't know how to pull the notes into the release. I have seen a way, but not sure how ell it works.

**Review by emoscardini — APPROVED** [review]

**Review by emoscardini — APPROVED** [review]

**Pull request merged**